main.js

```diff
@@ -66,19 +67,19 @@ function deleteCard() {
   var target = event.target.parentNode.parentNode;
   for (var i = 0; i < ideas.length; i++) {
     if (ideas[i].id === parseInt(target.id)) {
-      ideas[i].deleteFromStorage();
+      localStorage.removeItem(ideas[i].id);
```

Comment:
I recall hearing that we didn't necessarily need to use the methods so this was a better way to implement the code in the main.js.